src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,50 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+
+        Manufacturer manufacturer = manufacturerService.get(2L);
```

Comment:
What if someone else runs your code, will they get the same result?

Reply:
Yes I understood. No, there will be no similar result. I have to add first manufacturer,  to get the result correctly. In my table I have result for this methods, but someone else dont have (if dont use my  before task, when i add manufacture for this id).